src/test/java/seedu/address/logic/commands/task/TlistCommandTest.java

```diff
@@ -28,7 +28,7 @@ public void setUp() {
     }
 
     @Test
-    public void execute_tlistIsSuccessfulShown_showsSameList() {
+    public void execute_tlistIsSuccessfullyShown_showsSameList() {
```

Comment:
Small English issue: changing to 'execute_tlistSuccessfullyShown_showsSameList()' would be more correct. (take out the Is)